### ISSUE TITLE
unnecessary mapping if balances are capped at 1

### ISSUE THREAD

**Issue by thegostep** [pokt-foundation/pocket-dao-voting-token#2]
https://github.com/pokt-foundation/pocket-dao-voting-token/blob/dc5424bdb3e5dfd80f98e5727e268c4742d55303/contracts/POKTDAO.sol#L27

**Comment by antron3000** [comment]
fair enough, but balanceOf has to return a number

**Comment by antron3000** [comment]
Yes you are right, we could change it to mapping (address=>bool), would save some gas.  Can do.

**Comment by antron3000** [comment]
done

**Comment by kaycoinsUSA** [comment]
> https://github.com/pokt-foundation/pocket-dao-voting-token/blob/dc5424bdb3e5dfd80f98e5727e268c4742d55303/contracts/POKTDAO.sol#L27

**Comment by antron3000** [comment]
hmm it looks like you are looking at a part of the commit history. The main branch has been updated.
